src/shared/api/schemas.ts

```diff
@@ -224,7 +225,7 @@ export const Transaction = z.object({
   block: z.number(),
   created_at: Timestamp,
   executable: z.enum(['Instructions', 'Wasm']),
-  status: z.enum(['Committed', 'Rejected']),
+  status: TransactionStatus,
 });
```

Comment:
For `DetailedTransaction`, use [`.extend`](https://zod.dev/?id=extend) from `Transaction`